### PR TITLE
lod=level of detail

### LINKED ISSUE #2256: LOD (Level Of Detail) should not be treated as an error

### PR DESCRIPTION
**Pull request by matkoniecz**
fixes #2256

I agree that it is not the most common acronym, but I prefer codespell to not have false positives (unless user opts-in)

### CODE REVIEW

**Comment by peternewman** [comment]
> I agree that it is not the most common acronym, but I prefer codespell to not have false positives (unless user opts-in)

Note that code is the one not included by default:
https://github.com/codespell-project/codespell/blob/618dda35977c58ecc7e8f4b57f6591315c2781df/codespell_lib/_codespell.py#L68

**Review by peternewman — APPROVED** [review]
LGTM, up to @matkoniecz if he wants to move it to code instead...

**Comment by matkoniecz** [comment]
> Note that code is the one not included by default:

I know. I put it into rare as I have seen it discussed in computer graphics/computer games outside of what is considered as programming.

**Comment by peternewman** [comment]
> > Note that code is the one not included by default:
> 
> I know. I put it into rare as I have seen it discussed in computer graphics/computer games outside of what is considered as programming.

Okay, that does mean the correction will still be offered by default in all cases (unless people explicitly turn off the rare dictionary).

**Comment by matkoniecz** [comment]
I think I opted out (or will) exactly because it may contain false positives.

File: codespell_lib/data/dictionary_rare.txt
Comment (peternewman) [inline codespell_lib/data/dictionary_rare.txt:94]: Actually a bit late (having just merged), but suggesting loud would also be good, as well as stopping any auto-fixing...

**Pull request merged**
